src/chainparams.h

```diff
@@ -34,6 +34,10 @@ struct CCheckpointData {
     double fTransactionsPerDay;
 };
 
+static const int DEFAULT_MAINNET_PORT = 8333;
+static const int DEFAULT_TESTNET_PORT = 18333;
+static const int DEFAULT_NOLNET_PORT = 9333;
```

Comment:
An enum is unsuitable here, and I do expect to find such definition in a header file.